### PR TITLE
Add registry credentials in aerospike cluster and operator helm charts

### PR DESCRIPTION
**Pull request by jfwm2**
This is useful for using a custom registry with credentials  to store aerospike server images

Signed-off-by: Jean-Francois Weber-Marx <jfwm@hotmail.com>
Signed-off-by: Jean-Francois Weber-Marx <jf.webermarx@criteo.com>

### CODE REVIEW

File: helm-charts/aerospike-cluster/templates/aerospike-cluster-rbac.yaml [hunk 1]
```diff
@@ -9,6 +9,12 @@ metadata:
     app: {{ template "aerospike-cluster.commonName" . }}
     chart: {{ .Chart.Name }}
     release: {{ .Release.Name }}
+{{- with .Values.imagePullSecrets }}
+imagePullSecrets:
+  {{- range . }}
+  - name: {{ . }}
+  {{- end }}
+{{- end }}
```
Comment (spkesan) [inline helm-charts/aerospike-cluster/templates/aerospike-cluster-rbac.yaml:17]: Can you add a commented out entry for `imagePullSecrets` in `values.yaml` with an example?

File: helm-charts/aerospike-kubernetes-operator/templates/rbac.yaml [hunk 2]
```diff
@@ -9,6 +9,12 @@ metadata:
     app: {{ template "aerospike-kubernetes-operator.fullname" . }}
     chart: {{ .Chart.Name }}
     release: {{ .Release.Name }}
+{{- with .Values.imagePullSecrets }}
+imagePullSecrets:
+  {{- range . }}
+  - name: {{ . }}
+  {{- end }}
+{{- end }}
```
Comment (spkesan) [inline helm-charts/aerospike-kubernetes-operator/templates/rbac.yaml:17]: Can you add a commented out entry for `imagePullSecrets` in `values.yaml` with an example?

**Review by spkesan — CHANGES_REQUESTED** [review]
Looks to be okay.
Can you make the requested changes so the PR is ready to be merged?

File: helm-charts/aerospike-cluster/templates/aerospike-cluster-rbac.yaml (hunk 1 above)
Comment (spkesan) [reply helm-charts/aerospike-cluster/templates/aerospike-cluster-rbac.yaml:17]: Also document the configuration in README.

File: helm-charts/aerospike-kubernetes-operator/templates/rbac.yaml (hunk 2 above)
Comment (spkesan) [reply helm-charts/aerospike-kubernetes-operator/templates/rbac.yaml:17]: Also document the configuration in README.

Comment (jfwm2) [reply helm-charts/aerospike-kubernetes-operator/templates/rbac.yaml:17]: Hello @spkesan I made the changes you requested in another commit; if it is ok for you, I will squash everything before the merge.

**Review by spkesan — APPROVED** [review]

**Pull request merged**